prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/converter/PMMLSignatureInterpreter.scala

```diff
@@ -4,6 +4,7 @@ import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
 
 object PMMLSignatureInterpreter {
 
+  // scalastyle:off
```

Comment:
😢

Reply:
needed here as we don;t want to turn this rule globally